codespell_lib/data/dictionary.txt

```diff
@@ -30196,6 +30284,8 @@ varients->variants
 varierty->variety
 variey->variety
 variing->varying
+varilue->variable
+varilues->variables
```

Comment:
suggest to add `value` and `values`

Reply:
→ 41160880037f606e0669b66ee584e865b4ed28d8